docs/send-data/hosted-collectors/http-source/zoom.md

```diff
@@ -0,0 +1,85 @@
+---
+id: zoom
+title: Zoom Source
+description: An HTTP Zoom Source is an endpoint for receiving traces.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/saas-cloud/zoom.png')} alt="Thumbnail icon" width="50"/><br/>
+
+Zoom HTTP Source is an endpoint for receiving Webhook events that are grouped into the following core event types:
+- Meeting Events
+- Webinar Events
+- Recording Events
+- Zoom Room Events
+- User Events
+- Account Events
+
+For more information on Zoom Webhook events, see this [Zoom web page](https://marketplace.zoom.us/docs/api-reference/webhook-reference).
+
+This section shows you how to configure Webhooks to collect events from Zoom. For more information, see [Create a Webhook-Only App](https://marketplace.zoom.us/docs/guides/getting-started/app-types/create-webhook-only-app).
+
+## Prerequisites
+
+When you create a Zoom Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/).
+
+Some Webhook events may not be available based on the plan type. Refer to the Prerequisite section for each Webhook event type on this [Zoom page](https://developers.zoom.us/docs/api/rest/reference/zoom-api/events/#operation/meeting.created) for account-created event types.
+
+## Setup and Configuration
+
+Follow the below steps to get the required fields for user configuration:
+1. Sign in to [Zoom Marketplace](https://marketplace.zoom.us/).
+1. In the upper right corner, click **Develop > Build App**.<br/><img src={useBaseUrl('img/send-data/zoom-sign-in.png')} alt="zoom-sign-in" width="400"/>
+1. **Create** a Webhook Only.
+1. Specify the following App Information:
+    - App Name    
+    - Short Description
+    - Company Name
+    - Developer Name
+    - Developer Email Address 
+  <br/><img src={useBaseUrl('img/send-data/zoom-app-info.png')} alt="zoom-app-info" width="500"/>
+1. Click **Continue**, to copy the **Secret Token** value. <br/><img src={useBaseUrl('img/send-data/secret-token.png')} alt="secret-token" width="500"/>
+
+## Create Zoom Source
+
+To configure an Zoom Source:
```

Comment:
Zoom HTTP Source?

Reply:
yes, it is HTTP source. I will update the title accordingly.